lib/rspec/support/ruby_features.rb

```diff
@@ -65,6 +65,12 @@ def supports_exception_cause?
         end
       end
 
+      def ripper_supported?
+        # JRuby supports Ripper but it seems to be unstable
+        # and also reports wrong line number on JRuby 9.0.0.0.
+        defined?(RUBY_ENGINE) && RUBY_ENGINE == 'ruby' && RUBY_VERSION >= '1.9.2'
```

Comment:
This is too bad about JRuby's ripper support not being good enough for us to use.  Has the issue been reported upstream?  I imagine @headius and co would want to know and do something about this.  Unfortunately, as this is currently written it'll lock JRuby users out from enjoying the benefits of the feature if/when JRuby's ripper is fixed.

Would it be possible to write some logic that detects if ripper is working on JRuby so that we allow it to be used if a user is running a future version of JRuby where the ripper issues have been fixed?  Although, the obvious ways to do that would involve loading ripper and testing it out, which I don't think we want to do...